docs/cli/pull-from-upstream.md

```diff
@@ -0,0 +1,56 @@
+---
+title: pull-from-upstream
+date: 2024-02-22
+sidebar_position: 8
+---
+
+# `packit pull-from-upstream`
+
+This is a detailed documentation for the update functionality of packit. The
+command creates a new pull request in Fedora (or CentOS Stream) using a selected or latest
+upstream release. This command is meant to be called from **dist-git repository**.
+
+
+## Requirements
+
+* Upstream git repository on GitHub/GitLab.
+* Upstream release (read, git tag) where version in spec file is equivalent to
```

Comment:
```suggestion
* Upstream release (both release and git tag) where version in spec file is equivalent to
```

Not sure if this is what you mean.

Reply:
I just mostly copied what we already have for `propose-downstream`, but in fact git tag should be enough from CLI